backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/address/AddressRestController.java

```diff
@@ -128,7 +128,7 @@ public JSONLookupValuesPage getAttributeTypeahead(
 
 		return addressRepo.getAddressDocumentForReading(docId)
 				.getFieldLookupValuesForQuery(attributeName, query)
-				.transform(page -> JSONLookupValuesPage.of(page, userSession.getAD_Language()));
+				.transform(page -> JSONLookupValuesPage.of(page, userSession.getAD_Language(),false));
```

Comment:
instead of adding `, false` everywhere, i think we should add a method `JSONLookupValuesPage.of` without this parameter and there delegate to `JSONLookupValuesPage.of(page, language, false)`